### ISSUE TITLE
Faster Casting gem shown as Culling strike on calcs

### ISSUE THREAD

**Issue by lixa4ik** [Openarl/PathOfBuilding#1643]
Recently I discovered that if you attach faster casting support gem and check out your Inc. Cast Speed attribute on calcs tab, there will be wrong source name Culling Strike in a table that shows up on mouse hovering. It was very annoying for me. I ended up disarming and deleting all skills and tree in order to find the source of this boost.
![image](https://user-images.githubusercontent.com/6255921/62834466-989a4580-bc55-11e9-9b3f-eaead9d00909.png)

Thank you in advance.

**Comment by lixa4ik** [comment]
And for some reason minion damage support gem isn't "minion" so that my buff from item +2 to socketed minion gems isn't accounting that. Dps is slightly lower than it should have been.
![image](https://user-images.githubusercontent.com/6255921/62835279-4b22d600-bc5f-11e9-9961-0e85ee3647e7.png)

**Comment by lixa4ik** [comment]
Same goes for "Minion and Totem Elemental Resistance". Minion tag missing as well.
![image](https://user-images.githubusercontent.com/6255921/62910578-b2d54000-bd89-11e9-9b76-227f8f87a5c8.png)

**Comment by hateborne** [comment]
I do not believe you are getting Culling Strike from Faster Casting. My proof is from a level 1, 0 points witch with only Fireball and Faster Casting. If you would drop a PoB Pastebin link, it would help debug.
<img width="703" alt="Screen Shot 2019-09-05 at 12 42 34 PM" src="https://user-images.githubusercontent.com/4691451/64361604-e1d28080-cfda-11e9-80a7-8dc0a7ab1c78.png">
<img width="1212" alt="Screen Shot 2019-09-05 at 12 43 19 PM" src="https://user-images.githubusercontent.com/4691451/64361622-ed25ac00-cfda-11e9-91cc-1815b0235126.png">

---

As for the minion gems, these appear resolved.
<img width="705" alt="Screen Shot 2019-09-05 at 12 45 36 PM" src="https://user-images.githubusercontent.com/4691451/64361794-41309080-cfdb-11e9-811f-8b5ce0a45157.png">
<img width="910" alt="Screen Shot 2019-09-05 at 12 45 57 PM" src="https://user-images.githubusercontent.com/4691451/64361800-442b8100-cfdb-11e9-8a67-ddb1a205ab3a.png">

---

Unless I'm missing something, this may be safe to close/cancel. :)

**Comment by lixa4ik** [comment]
Take a look. We have summon skeleton with faster casting attached to it. Unless faster casting gem has 2+ quality it has right tooltip. We give 2 or more quality and it changes to culling strike.
![image](https://user-images.githubusercontent.com/6255921/64488580-56fbbb00-d252-11e9-9e3a-3f0c9c4ce45f.png)
Take a look at pastebin. The issue is present:
https://pastebin.com/dtBQuaWa

**Comment by hateborne** [comment]
> Take a look. We have summon skeleton with faster casting attached to it. Unless faster casting gem has 2+ quality it has right tooltip. We give 2 or more quality and it changes to culling strike.
> Take a look at pastebin. The issue is present:
> https://pastebin.com/dtBQuaWa

@lixa4ik I am unable to reproduce this on 1.4.149 or 1.4.151. I took the link you provided, changed the Faster Casting gem quality to 0, 1, 2, and 3 (checking each time). I checked both with and without 'Show Minion Stats' on the Calcs tab. I checked each individual Skeleton Caster variant (Fire/Cold/Light), as well as checking the base 'Summon Skeleton' melee type. None of this added Culling Strike. 

Can you confirm you are running 1.4.151? If so, did you manually compile or download a binary (precompile) version? Lastly, can you provide a screenshot of the app as a whole (with Calcs tab 'in focus')? There is nothing that I have tried that can reproduce this on two different installs of the app (on two laptops). :(

**Comment by lixa4ik** [comment]
Hey there! Been a while. I have noticed that after saving your build to pastebin the issue with wrong tooltip seems to be resolved. So here is the recipe to reproduce the issue on version 1.4.152:
1) Get fresh latest release build from github v1.4.147 (zip version). It will download update files and on your first load it will be 1.4.152
2) create fresh build
3) go to skills and create new group
4) add Summon Skeletons 20/0
5) add Faster Casting 21/0
5) Go check Calcs tab and check "Inc. Cast Speed" in Attack/Cast Rate tab. It says Culling strike.
![image](https://user-images.githubusercontent.com/6255921/68455680-003c3e80-020d-11ea-958c-d716b774a4d6.png)

I hope you will be able to replicate this now!
